### PR TITLE
Simplify Profile

### PR DESCRIPTION
**Pull request by seanpdoyle**
https://trello.com/c/Oz6iPKen

This commit implements the design changes introduce in [5b4243c][].

It removes the `donations/current` partial in favor of a
`donations/donation` partial. Tests covering `donations/current` are
modified to provide coverage for `donations/donation`.

There are still unimplemented portions of the `/profile` page. They're
copied, unchanged, from the `high_voltage` version of `/profile` and
will be implemented in the future.

[5b4243c]: https://github.com/thoughtbot/freshfoodconnect/commit/5b4243c8da7be93865eeac3530d7280063813351

**Pending**

screen shot 2016-04-12 at 6 30 37 pm

**Confirmed**

screen shot 2016-04-12 at 6 31 32 pm

**Declined**

screen shot 2016-04-12 at 6 31 57 pm

**Edit**

screen shot 2016-04-12 at 6 37 51 pm

### CODE REVIEW

**Comment by corwinharrell** [comment]
@seanpdoyle screenshot or does everything look pretty much the same?

File: spec/features/donor_edits_location_spec.rb [hunk 1]
```diff
@@ -8,7 +8,6 @@
     new_pickup_location = {
       address: "123 Fake Street Denver CO",
       zipcode: zone.zipcode,
-      notes: "on my porch",
     }
```
Comment (desi) [inline spec/features/donor_edits_location_spec.rb]: Are we removing the notes section?

**Comment by desi** [comment]
I just had a question about the notes and whether or not that was being removed. (I thought we did want a way for the supplier to leave a note about location of the food).

File: spec/features/donor_edits_location_spec.rb (hunk 1 above)
Comment (seanpdoyle) [inline spec/features/donor_edits_location_spec.rb]: @desi temporarily.

I've made [a note on the card](https://trello.com/c/Oz6iPKen/78-as-a-supplier-i-want-to-be-able-to-log-into-the-app-and-view-my-profile#comment-570d75cb2e569299dd8b6f79).

The idea is that it's not currently working properly, and needs some design. My options were either to remove it and get tests passing, make the tests pass by injecting it into the page without regard for design, or block the PR until the design is ready.


Do you think we should keep it around?

**Comment by seanpdoyle** [comment]
@corwinharrell I've attached screenshots.

The `Donating` and `Not Donating` buttons are a bit wonky because the designs were done with `<a>` tags instead of `<button>` tags, while the implementation has `<button>` tags.

Using buttons here gives our page more semantic meaning and accessibility.

How would you like to proceed with fixing the design?

**Comment by corwinharrell** [comment]
@seanpdoyle go ahead and merge and I'll take care of that and the notes in the sidebar in another PR :+1:

File: spec/features/donor_edits_location_spec.rb (hunk 1 above)
Comment (seanpdoyle) [inline spec/features/donor_edits_location_spec.rb]: @desi it looks like Corwin's going to handle it soon: https://github.com/thoughtbot/freshfoodconnect/pull/42#issuecomment-209132508.

Good to merge?

Comment (desi) [inline spec/features/donor_edits_location_spec.rb]: Yep sounds good. Thanks for the explanation.

**Comment by desi** [comment]
:+1:

**Pull request merged**